test/soca/create_obsdb.py

```diff
@@ -37,5 +37,34 @@
                                        'salt_profile_fnmoc',
                                        'icec_EMC',
                                        'icefb_GDR']})
+    ufsda.r2d2.store(obsstore)
 
+# Create the test R2D2 database for output from bufr2ioda tests
```

Comment:
You don't need to redefine the entire dictionary, it's almost the same as the one above bar a few keys.
Maybe something like this instead:
```python
obsstore['source_file_fmt'] = '{source_dir}/{obs_type}_{year}{month}{day}.nc'
obstore['obs_types'] = ...
```

Reply:
done